### PR TITLE
Node and Vagrant evolves

### PR DESCRIPTION
**Pull request by camlafit**
These commits solves some problem to run a Vagrant instance
* first we an unique git clone
* update to last LTS to solve pb avec #187 

Second part is an upgrade node environment to reduce some Error and Warning during installation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,12 +86,12 @@
     "grunt-filerev": "~2.3.1",
     "grunt-inline-angular-templates": "~0.1.5",
     "grunt-line-remover": "~0.0.2",
-    "grunt-mocha-test": "~0.12.7",
+    "grunt-mocha-test": "~0.13.2",
     "grunt-replace": "~1.0.1",
     "grunt-usemin": "~3.1.1",
-    "grunt-webfont": "~1.4.0",
+    "grunt-webfont2": "~1.0.0",
```
Comment (gmetais) [inline package.json]: And why switching to grunt-webfont2? The grunt-webfont project seems to be maintained.

**Comment by gmetais** [comment]
Hi @camlafit,

Thanks a lot for this pull request 👍. I've just commited some of your changes into the develop branch and it will make it's way to the master branch soon:
- https://github.com/gmetais/YellowLabTools/commit/bb3a94eb844ce186e3010b85237a054c0f6ab7bb : Use a synced folder in Vagrant
- https://github.com/gmetais/YellowLabTools/commit/49b460eee3c4b78200858e589e96386c7bd2c501 : Update the vagrant box and Node version

I'm keeping this pull request open, cause I'm still interested by your other changes if they can fix something I didn't see!

**Comment by camlafit** [comment]
Hi

I saw an another problem with jenkins check. Node version are different. I
presume we can correct this also. I'm in late about this.
I hope provide an add soon.

**Comment by camlafit** [comment]
About package.json is just changed to follow warning get by npm install.
tree requirement plugins is a bit complex , de mémoire, nan is
required by a third or fourth level plugin

**Comment by gmetais** [comment]
Thanks @camlafit, I'll integrate this directly in YLT v2's branch!

**Comment by camlafit** [comment]
Hello 

Sorry it's again in WIP. I've tried to move to more recent boxes, is not yet ok.

**Pull request closed without merging**